packages/gatsby-source-wordpress/__tests__/process-node.test.js

```diff
@@ -1,5 +1,7 @@
 import execall from "execall"
 
+import { replaceNodeHtmlImages } from "../dist/steps/source-nodes/create-nodes/process-node"
```

Comment:
iirc it was because there's a mix of TS & JS and source-wordpress has it's own babel config - so this runs the tests with the code exactly how the user runs it.